errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/DataBindingUtil.java

```diff
@@ -351,7 +351,8 @@ private static void assertTypeIsDataBinder(final MetaClass type) {
    *          the type to check
    */
   private static void assertTypeIsBindable(final MetaClass type) {
-    if (!type.isAnnotationPresent(Bindable.class) && !getConfiguredBindableTypes().contains(type)) {
+    if ((!type.isAnnotationPresent(Bindable.class) && !getConfiguredBindableTypes().contains(type))
```

Comment:
Since this logic is getting more complex, it probably makes sense to reuse `isBindableType` here.

Reply:
Ok